.circleci/config.yml

```diff
@@ -40,6 +40,19 @@ workflows:
             tags:
               only: /^v.*/
 
+      - architect/push-to-docker:
+          context: "architect"
+          name: push-to-docker
+          image: "docker.io/giantswarm/aws-operator"
+          username_envar: "DOCKER_USERNAME"
+          password_envar: "DOCKER_PASSWORD"
+          requires:
+            - go-build
+          # Needed to trigger job also on git tag.
+          filters:
+            tags:
+              only: /^v.*/
+
```

Comment:
this was missing and causing me troubles with testing as it took long until crsync made it available in docker.io